bridge/ApiRequestHandler.cpp

```diff
@@ -588,6 +587,12 @@ httpd::Response ApiRequestHandler::handleAbout(const httpd::Request& request,
         response._headers["Content-type"] = "text/json";
         return response;
     }
+    else if (utils::StringTokenizer::isEqual(nextToken, "capabilities"))
+    {
+        httpd::Response response(httpd::StatusCode::OK, "[\"SMB_API.1\", \"SSRC_REWRITING.1\"]");
```

Comment:
Please use camel case for these capabilities in the response body.
This just makes it harder to read and eyes ache.

Reply:
As agreed with @olofkallander , I made it lowercase in SFU, but with "_" to facilitate robust parsing in CS.